en/docs/tutorials/integrating-with-solace.md

```diff
@@ -1,168 +1,76 @@
 # External Broker and Gateway Integration with API Manager
 
-WSO2 API Manager now supports integrating external third party Gateways and Brokers. Solace is an advanced event broker which is efficiently streaming events and information across cloud, on-premises, and IoT environments. In this section, you will integrate the Solace gateway environment with API Manager.
+Solace is an advanced event broker which efficiently streams events and information across cloud, on-premises, and IoT environments. In this section, you will integrate the Solace gateway environment with API Manager.
```

Comment:
We talk about the solace gateway integration with WSO2 API Manager here in the heading and main description, but the content makes it a little bit unclear as we do not talk about a gateway anywhere else. We only refer to it as a Solace Event API. May be we can write a single sentence at the beginning to mention the connection between a Solace Event API and Solace Gateway. WDYT?

Reply:
It's in the second paragraph next to this. 


> A **Solace Event API** will be represented as an API in WSO2 API Manager. This API can be subscribed by applications in WSO2 Developer Portal, and the tokens generated by the Developer Portal applications can be used to authenticate and authorize requests to the Solace Event Broker via the Solace Event API.